nnpdf_data/theory.py

```diff
@@ -0,0 +1,59 @@
+"""
+This module provides an unique source and definition for all the possible parameters
+that a theory card can contain. 
+It also implement some utilities function to manage the theory cards. 
+"""
+
+from dataclasses import dataclass
+
+
+@dataclass(frozen=True)
+class _TheoryCard:
+    ID: int
+    PTO: int
+    FNS: str
+    DAMP: int
+    DAMPPOWERc: None #Power of the damping factor in FONLL for the c
+    DAMPPOWERb: None #Power of the damping factor in FONLL for the b
+    IC: int
+    IB: int
+    ModEv: str
```

Comment:
This I am not sure is used anymore given that EKO gets this through the operator card (@felixhekhorn please confirm this) and in pineko this field is not read at all. It might be still nice to have just to remember.

Reply:
as we said: we should change pineko to propagate the information coming from here to the observable card (overwriting whatever is set there)